bee-common/bee-common/src/packable/io.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+extern crate std;
+
+use super::{Packer, Unpacker};
+
+use std::io::{self, Read, Write};
+
+impl<W: Write> Packer for W {
+    type Error = io::Error;
+
+    fn pack_bytes(&mut self, bytes: &[u8]) -> Result<(), Self::Error> {
```

Comment:
I wonder if we'll ever have the need for async pack/unpack, like async versions of Read/Write 🤔

Reply:
![image](https://user-images.githubusercontent.com/31802960/118665912-5b62d000-b7b8-11eb-888f-a4fea9ea7c94.png)